text/XXXX-delete_and_drop.md

```diff
@@ -0,0 +1,225 @@
+- Feature Name: delete and drop
+- Start Date: 2020-05-02
+- RFC PR: [#53](https://github.com/qri-io/rfcs/pulls/53)
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Rename the remove command to `delete`, and clarify it's purpose is for deleting versions from a dataset history. Create a new command `drop` for removing datasets from remotes.
+
+# Motivation
+[motivation]: #motivation
+
+The _user experience (UX)_ of removing data from Qri needs some work. We're seeing a number of users get into states they feel they cannot recover from, and rather than use tools like `qri remove`, they're deleting repositories entirely in an effort to start over. 
+
+A better UX for removing data from qri should clearly answer two common "uh oh" scenarios a user is likely to encounter:
+
+1. "I made a mistake while saving"
+2. "I want to undo a push"
+
+The first is an example of removing data from history. The second is an example of removing data from the network. In both scenarios the user is a data author, and a good answer to these questions should present well-documented, easy-to-find commands that when executed leave the user feeling that they're in a clean state. 
+
+A well-documented desctrive command should be clearly labeled as the "opposite" of a constructive command and present warnings _before_ execution. Once executed, commands remove data should show as few caveat/errors/warnings as possible, or at least connect caveats to a clear next course of action. A feeling of "clean state" comes when vommands like `qri log`, `qri list` and `qri status` match user expectations after running a destructive command.
+
+To improve our "undo UX", this RFC proposes three changes to qri CLI:
+1. rename `remove` to `delete`, clarify its text to talk about history instead of just storage
```

Comment:
I'm fine with this change. Though not 100% convinced it's necessary, I would lean towards thinking that `delete` is a stronger opposite to `save` than `remove`.